modules/drivers/redshift/test/metabase/test/data/redshift.clj

```diff
@@ -80,7 +80,10 @@
 
 ;;; Create + destroy the schema used for this test session
 
-(defn- execute! [format-string & args]
+(defn execute!
+  "Executes SQL statements against the current test session's schema. format-string is the statement(s) (formatted)
+   and any format args follow."
+  [format-string & args]
```

Comment:
Rather than making functions public for the sake of tests, it's better to use the var form to call them in your tests e.g. `#'redshift/execute!`

https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#make-things-private

Reply:
keeping stuff private makes it easier to refactor stuff in the future -- if you want to change how `redshift/execute!` works you don't need to go around grepping the codebase to try to figure out if anybody is using it (tests, at least, should fail if you change it) -- the same reasoning applies to the "use consistent namespace aliasing" rule

Reply:
Changed back to private and used this approach for invoking instead.